db/seeds.rb

```diff
@@ -34,18 +34,3 @@
 following.each { |followed| user1.follow(followed) }
 followers.each { |follower| follower.follow(user1) }
 
-49.times do |n|
-  title  = Faker::Book.title
-  des = "Lorem Ipsums"
-
-  Category.create!(title:  title,
-                  description: des)
-end
```

Comment:
@johnedelmapa Don't remove category seeders. This is very helpful for testing

Reply:
Noted thank you for the info.